scripts/run_wheeler.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-CONTAINER_NAME="das-node-wheel"
+CONTAINER_NAME="das-node-wheeler"
 
 docker run \
     --name=$CONTAINER_NAME \
```

Comment:
```suggestion
docker run \
    --name=$CONTAINER_NAME \
    -e _USER=$(id -u) \
    -e _GROUP=$(id -g) \
```

Reply:
With `--rm` added above, you can remove the last two lines (`sleep ... docker rm ...`).